frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -32,14 +34,24 @@ const MultiColumnFieldRow: React.FC<MultiColumnFieldRowProps> = ({
   isReadOnly,
   disableDeleteRow,
   spans,
+  complexFields = [],
 }) => {
   const { t } = useTranslation();
   return (
-    <div className="odc-multi-column-field__row">
+    <div
+      className={cx('odc-multi-column-field__row', {
+        'odc-multi-column-field__row--singleLine': !complexFields,
```

Comment:
Should we use a better check than `!complexFields`?

Reply:
Agreed. This will always result in `false`. Maybe you need to use `_.isEmpty(complexFields)` here.

Reply:
Good point, thanks guys! Easy update.